modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -25,52 +25,15 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
     "status": 500
   }.freeze
 
-  swagger_path '/notice_of_disagreements/contestable_issues' do
-    operation :get, tags: NOD_TAG do
-      key :operationId, 'getNODContestableIssues'
-
-      key :summary, 'Returns all contestable issues for a specific veteran.'
-
-      description = 'Returns all issues associated with a Veteran that have' \
-      'not previously been decided by a Notice of Disagreement' \
-      'as of the `receiptDate`. Not all issues returned are guaranteed to be eligible for appeal.' \
-      'Associate these results when creating a new Notice of Disagreement.'
-      key :description, description
-
-      parameter name: 'X-VA-SSN', 'in': 'header', description: 'veteran\'s ssn' do
-        key :description, 'Either X-VA-SSN or X-VA-File-Number is required'
-        schema '$ref': 'X-VA-SSN'
-      end
-
-      parameter name: 'X-VA-File-Number', 'in': 'header', description: 'veteran\'s file number' do
-        key :description, 'Either X-VA-SSN or X-VA-File-Number is required'
-        schema type: :string
-      end
-
-      parameter name: 'X-VA-Receipt-Date', 'in': 'header', required: true do
-        desc = '(yyyy-mm-dd) In order to determine contestability of issues, ' \
-          'the receipt date of a hypothetical Decision Review must be specified.'
-        key :description, desc
-
-        schema type: :string, 'format': :date
-      end
-
-      key :responses, read_json_from_same_dir['responses_contestable_issues.json']
-      security do
-        key :apikey, []
-      end
-    end
-  end
-
-  swagger_path '/decision_reviews/notice_of_disagreements' do
+  swagger_path '/notice_of_disagreements' do
     next unless PATH_ENABLED_FOR_ENV
 
     operation :post, tags: NOD_TAG do
       key :summary, 'Creates a new Notice of Disagreement.'
 
       description = 'Submits an appeal of type Notice of Disagreement.' \
-      'This endpoint is analogous to submitting VA Form 10182' \
-      'via mail or fax directly to the Board of Veterans’ Appeals.'
+      'This endpoint is the same as submitting [VA Form 10182](https://www.va.gov/vaforms/va/pdf/VA10182.pdf)' \
```

Comment:
Does swagger support markdown this way? Huh. Had no idea!

Reply:
It does indeed :)